File: javascript/problem020.js
var numberArray = [1];

function calculate(a){
    for(j=0; j<numberArray.length; j++){
      numberArray[j] = numberArray[j] * a;
    }
    for(j=0; j<numberArray.length; j++){
      if(numberArray[j] >= 10){
        if(numberArray[j+1] == undefined){
          numberArray[j+1] = parseInt(numberArray[j] / 10);
        }
        else{
          numberArray[j+1] += parseInt(numberArray[j] / 10);
        }
        numberArray[j] = numberArray[j] % 10;
      }
    }
}

function getValue(){
  var value = 0;
  for(i=0; i<numberArray.length; i++){
    value = value + numberArray[i];
  }
  return value;
}


for(a=2; a<100; a++){
  calculate(a);
}

var result = getValue();

console.log(result);

File: java/src/Problem020.java
import java.math.BigInteger;

/**
 * Created by 김 on 2016-02-05.
 */
public class Problem020 {

    int result(){
        BigInteger value = new BigInteger("1");
        for(int i=2; i<=100; i++){
            value = value.multiply(BigInteger.valueOf(i));
        }
        String str = value.toString();
        int resultValue = 0;
        for(int j=0; j<str.length(); j++){
            resultValue = resultValue + Integer.parseInt(str.substring(j,j+1));
        }
        return resultValue;
    }

    public static void main(String args[]) {
        Problem020 problem = new Problem020();
        System.out.println(problem.result());
    }
}
